set-search-api-client/src/main/java/eu/europeana/set/search/service/impl/SearchApiClientImpl.java

```diff
@@ -181,6 +179,27 @@ JSONObject searchItems(String uri) throws SearchApiClientException {
 	}
     }
 
+    /* (non-Javadoc)
+     * @see eu.europeana.set.search.service.SearchApiClient#validateUrl(java.lang.String, java.lang.String)
+     */
+    public int validateUrl(String uri, String apiKey) throws SearchApiClientException {
```

Comment:
not needed, update searchItems method to handle responses with 0 items included

Reply:
searchItems already handles it with initial value -1